data/human/transport missions.txt

```diff
@@ -3594,6 +3595,747 @@ mission "Hauler VI cargo prototype 3"
 			`	Veronica disembarks, walks over to you, and apologizes for the difficult trip. She explains that the cooling systems failed, causing the sushi calamity you experienced during the multiple legs of your journey.`
 			`	You are paid <payment> and she asks you not to talk about this escort job to anybody. You leave the docks quickly in order to escape the horrible smell of overheated fish that is coming from Veronica and her ship.`
 
+
+mission "Family back to Paradise - no luxury"
+	name "Take family home to <planet>"
+	description "A family worried about the potential destruction in the Free Worlds wants to cut their vacation short and go home to <destination>."
+	source
+		government "Free Worlds"
+		attributes "tourism"
+	destination "Mainsail"
+	passengers 3
+	cargo "personal belongings" 1
+	to offer
+		has "FW Katya 7C: done"
+	to fail
+		has "free worlds plot completed"
+	on offer
+		require "Luxury Accommodations" 0
+		conversation
+			`After walking around the spaceport for a while, you head back to your ship to find a middle-aged couple and two grade-school girls (probably their daughters, you reason) waiting for you at your hanger. While the man tries to resolve an argument between the two girls, the lady notices you immediately and begins to approach you. "Thank goodness you're here," she says. "We've been going from ship to ship looking for transport back home to the Paradise Planets. This war has caught us right in the middle of our vacation and we can't risk being anywhere near here when the Navy attacks, especially with our little angels," she gestures toward her daughters, now in a full-blown fight. "If you can find the time to take us, we will pay you very well."`
+			choice
+				`	"Sure, I'll take you."`
+				`	"Okay, but that's pretty far. This better be worth it."`
+				`	"No, I don't think so. Sorry."`
+					decline
+			`	"Then it's a deal," she shakes your hand firmly. The whole family gets onto the <ship> without a word, leaving you to load their many bags and cases.`
+				accept
+
+	on complete
+		payment 66999
+		conversation
+			`You're not sure if you have ever met a more pretentious and entitled family. The girls fought non-stop and they all complained constantly of the lack of luxury accommodation on your ship. The lady gives you <payment> and they leave without a word. They never even told you their names. You are glad to finally be free of them. Although you feel you should have been paid more for how terrible they were.`
+
+	on visit
```

Comment:
Mission structure should follow the style of existing missions. That is, on visit should go above on complete.

Reply:
Thanks for looking it over. I will work on the style changes this coming week. And my apologies. I somehow missed those suggestions in the hidden conversations. The I will implement the ones that were to old to commit. Im not sure exactly what you mean about eating the white space but i assume it has to do with when i switched to atom as my text/code editor, which messed with the spacing with I first used it. I thought I got it sorted. I will give it another look.  And finally, part of me had given up on this story, mostly because a couple people told me it was very unlikely to make it into the game. I assume since others are still making corrections there is still a chance? I had started writing a "neutral" / "guildish" storyline instead. Any advice for which of the two ideas is more productive for the game (and thus more likely to be included)?